website/project/views/contributor.py

```diff
@@ -365,7 +366,12 @@ def get_timestamp():
 
 
 def throttle_period_expired(timestamp, throttle):
```

Comment:
Looks like this doesn't have any tests. Could you add a few? I wouldn't go crazy, but cover the major cases.